product_multi_category/models/product.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2009 Akretion,Guewen Baconnier,Camptocamp,Avanzosc,Sharoon Thomas,Sodexis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    categ_ids = fields.Many2many(
+        comodel_name="product.category",
+        relation="product_categ_rel",
+        column1="product_id",
+        column2="categ_id",
+        string="Extra Categories",
```

Comment:
Instead of extra categories my suggestion is to change the string to multi categories.

Reply:
IMHO that would make sense if we just had a single field for all the categories. As they are separated between `Product Category` and  `Extra Categories`, I'd say it makes sense to keep it this way.